### DISCUSSION TITLE
Deploy new k3s cluster, keep existent Load Balancer

### DISCUSSION THREAD

**Discussion by rmelilloii** [vitobotta/hetzner-k3s#253] [Q&A] [answered]
Hello and good afternoon!

I have a common request form my customers to keep our endpoints with static IP. As the process to whitelist them is very timing consuming.

In some cases we need to delete our cluster and create a new one with all its services. I am having trouble to then, force the new cluster to use an existing hetzner LB (so I can use a known Public IP).

Is there any suggestion or support to inform the name/IP of an existent Hetzner LB?
Any suggestion is welcome.

Thanks.

**Comment by vitobotta** [comment]
HI :)

As far I as I know there is no way to "reserve" an IP address in Hetzner Cloud. But I'd try to confirm with support to be sure.

**Reply by rmelilloii** [reply]
Hello! Thanks for your reply. Yes, in hetzner, there is no way to do that at the moment. Can I simply install the ingress-controller via helm informing the "external-IP" parameters and it should work? Work = Give me a green status on my Hetzner LB.

Thanks.

**Comment by derlinuxer** [comment, marked as answer]
I guess i should be possible if you mean the lb for services. In ingress controller config you can set the name of the lb, if its already exist this one will used. I think I did that before. I also lock my lbs to prevent deleting via hetzner api  when delete or reconfigure kube components.
If it's time I'll try it tomorrow.

**Reply by derlinuxer** [reply]
Jap it's working. Create a clean / empty / locked lb and set it's name in ingress controller config. The lb has been filled with settings on helm install ingress controller.

**Reply by rmelilloii** [reply]
Nice! So if it is an old/existent LB it will behave similarly. I will validate before applying in Prod, but thanks for the suggestion/test/validation. I appreciate.
